java/streaming/src/main/java/com/ray/streaming/core/graph/ExecutionNode.java

```diff
@@ -0,0 +1,95 @@
+package com.ray.streaming.core.graph;
+
+import com.ray.streaming.core.processor.StreamProcessor;
+import com.ray.streaming.plan.VertexType;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExecutionNode implements Serializable {
```

Comment:
Can you provide some information on the ExecutionNode? Is it an actor that executes Ray tasks?

Reply:
yes, an executionNode will run on an actor, such as map/filter. and we use ray.call to implement send message to downstream